frontend/www/js/controllers/playfestController.js

```diff
@@ -22,4 +17,18 @@ playfestApp.controller('playfestController', ['TalkToBackendService', '$scope',
     $scope.confirmationData = data;
   }
 
+  function _makeHash(playlistName, arrayOfArtists){
+    return { "playlistName" : playlistName, "arrayOfArtists" : arrayOfArtists };
+  }
+
+  function _checkIfNamed(playlistName, data){
+    if (playlistName == null){
+      $scope.confirmationData = "You must enter a playlist name";
+    } else {
+      $scope.confirmationData = null;
```

Comment:
maybe put something like 'creating your playlist, please wait', to give feedback it has been actually clicked?